src/plugins/data/public/search/aggs/agg_types.ts

```diff
@@ -86,7 +88,24 @@ export function getAggTypes(deps: { uiSettings: IUiSettingsClient }) {
       ipRangeBucketAgg,
       termsBucketAgg,
       filterBucketAgg,
-      getFiltersBucketAgg({ uiSettings }),
+      getFiltersBucketAgg({
```

Comment:
can you please move this into `buckets/filters` and `buckets/adjacency_matrix` and move the common code thats now in `buckets/filters` into `buckets/lib/filters`